src/modules/modals/Import.vue

```diff
@@ -315,21 +327,26 @@ export default {
 				if (res.status === 200) {
 					this.preview = res.data
 					this.loading = false
-				} else if (res.status === 401) {
-					console.debug('error while importing', res)
-					showError(t('tables', 'Could not import, not authorized. Are you logged in?'))
-				} else if (res.status === 403) {
-					console.debug('error while importing', res)
-					showError(t('tables', 'Could not import, missing needed permission.'))
-				} else if (res.status === 404) {
-					console.debug('error while importing', res)
-					showError(t('tables', 'Could not import, needed resources were not found.'))
 				} else {
-					showError(t('tables', 'Could not import data due to unknown errors.'))
-					console.debug('error while importing', res)
+					if (res.status === 401) {
```

Comment:
Seems like some code duplication, maybe we can move the error handling into a shared method?